tests/rust_usage_test/tests/integration_test.rs

```diff
@@ -1716,7 +1738,14 @@ mod follow_impls {
 
         let vec: Vec<u8> = vec![255, 255, 255, 255, 1, 2, 3, 4];
         let off: flatbuffers::FollowStart<&FooStruct> = flatbuffers::FollowStart::new();
-        assert_eq!(*off.self_follow(&vec[..], 4), FooStruct{a: 1, b: 2, c: 1027});
+        #[cfg(target_endian = "little")]
+        {
+            assert_eq!(*off.self_follow(&vec[..], 4), FooStruct{a: 1, b: 2, c: 1027});
+        }
+        #[cfg(not(target_endian = "little"))]
+        {
+            assert_eq!(*off.self_follow(&vec[..], 4), FooStruct{a: 1, b: 2, c: 772});
```

Comment:
Is the idea with this that you've identified a case where the data are not read correctly, and you are encoding it in a test?

Reply:
This is just an endianness issue. This test (and the `to_slice_of_struct_elements` one below) just manually creates a `FooStruct` from a vector of bytes, and the `FooStruct.c` field has a different representation on different platforms since it's a two-byte field. On a little-endian machine, `[3, 4]` will cast into an `i16` as `(4 << 8) + 3 = 1027` (the lowest address in memory being the least-significant byte). On a big-endian machine, `[3, 4]` will cast into an `i16` as `(3 << 8) + 4 = 772` (the lowest address in memory being the most-significant byte).

Reply:
This is superseded by other other conversation in this review.